rts/Lua/LuaVAOImpl.cpp

```diff
@@ -185,6 +194,116 @@ int LuaVAOImpl::AddObjectsToSubmissionImpl(const sol::stack_table& ids)
 	return submitCmds.size() - idsSize;
 }
 
+template<typename TObj>
+void LuaVAOImpl::Bins::ModifyImpl(const sol::stack_table& removedObjects, const sol::stack_table& addedObjects,
+	sol::optional<size_t> removedCount, sol::optional<size_t> addedCount,
+	std::vector<SDrawElementsIndirectCommand>& submitCmds)
```

Comment:
This should belong to the class and accessed directly, no need to carry it along with arguments.

Reply:
> This should belong to the class and accessed directly, no need to carry it along with arguments.

The thing is, submitCmds is the only thing that is used from LuaVAO here. All the other fields are good to work this way.
Maybe move this thing in another cpp altogether?